htmlslacker/htmlslacker.py

```diff
@@ -66,14 +70,17 @@ def handle_endtag(self, tag):
             self.output += '>'
         if tag == 'code':
             self.output += '`'
+        if tag == 'style' or tag == 'script':
```

Comment:
I think it would be better to use `else`

Reply:
@sebbacon

Reply:
Not sure what you mean - if you mean any tag that is not explicitly handled should result in the text content being dropped, that would mean `h1` text would get dropped, for example, or anything in a `<span>`, or things in any semantic tags like `<header>` or whatever? 

Or if you mean these should all be `elif` switches, then sure :)

Reply:
Yes, I meant any tag which is not supported by slack should be dropped, but maybe it is not correct, better to output only content.